src/aiotk/tcp.py

```diff
@@ -3,8 +3,24 @@
 
 import asyncio
 
+from asyncio import Future  # noqa: F401
+from asyncio import StreamReader
+from asyncio import StreamWriter
+from asyncio import Task  # noqa: F401
```

Comment:
**Note to self**: prior to Python 3.6, variables types must be declared in comments.  `mypy` requires that we import them, but `flake8` complains that they are not used.  We must resort to telling `flake8` to ignore these types.

Futhermore, it seems like `#noqa: F401` cannot be applied in block imports (neither individually, nor for the entire block), so we must expand the blocks and explicitly silence each type.

Reply:
Given that Python 3.6 supports type annotations for variables, there might not be anything to fix.  I guess code bases that support Python 3.5 are somewhat stuck with this workaround.